jekyll/_cci2/github-apps-integration.adoc

```diff
@@ -121,7 +121,10 @@ There are a number of built-in environment variables that are not available in G
 
 CircleCI's GitHub App integration provides fine-grained permissions, uses short-lived tokens, and gives you more control over which repositories CircleCI has access to.  The CircleCI GitHub App also enables functionality like link:https://discuss.circleci.com/t/trigger-pipelines-from-anywhere-inbound-webhooks-now-in-preview/49864[inbound webhooks], link:https://discuss.circleci.com/t/circleci-config-suggestions-bot/47918[CircleCI's config suggestions bot], and the ability to use link:https://discuss.circleci.com/t/product-update-multiple-configuration-files-per-project/50637[multiple configuration files within one project].
 
-There is no automated way to migrate your organization from the GitHub OAuth app to CircleCI's GitHub App integration.  Before attempting to move your information from an org integrated with the GitHub OAuth app to an org integrated with CircleCI's GitHub App, **confirm that you do not immediately need any of the functionality listed in the <<currently-not-supported>> section below**. If you cannot move your organization because of missing functionality, link:https://docs.google.com/forms/d/e/1FAIpQLSfnYhFLjmZ0OP8goemexAvgHDPJqgHyDF1QiIl2HdPktTKvlQ/viewform[tell us why].  If you would like to switch from the OAuth app integration to the GitHub App integration, follow these steps:
+There is no automated way to migrate your organization from the GitHub OAuth app to CircleCI's GitHub App integration.  Before attempting to move your information from an org integrated with the GitHub OAuth app to an org integrated with CircleCI's GitHub App, **confirm that you do not immediately need any of the functionality listed in the <<currently-not-supported>> section below**. If you cannot move your organization because of missing functionality, link:https://docs.google.com/forms/d/e/1FAIpQLSfnYhFLjmZ0OP8goemexAvgHDPJqgHyDF1QiIl2HdPktTKvlQ/viewform[tell us why]. 
+ If you have a dedicated account team at CircleCI, please contact them first.   
+
+If you would like to switch from the OAuth app integration to the GitHub App integration, follow these steps:
```

Comment:
```suggestion
There is no automated way to migrate your organization from the GitHub OAuth app to CircleCI's GitHub App integration.  Before attempting to move your information from an org integrated with the GitHub OAuth app to an org integrated with CircleCI's GitHub App, **confirm that you do not immediately need any of the functionality listed in the <<currently-not-supported>> section below**.  If you have a dedicated account team at CircleCI, contact them first to discuss your migration.    

If you cannot move your organization because of missing functionality, link:https://docs.google.com/forms/d/e/1FAIpQLSfnYhFLjmZ0OP8goemexAvgHDPJqgHyDF1QiIl2HdPktTKvlQ/viewform[tell us why].

If you would like to switch from the OAuth app integration to the GitHub App integration, follow these steps:
```